astropy/io/fits/tests/test_image.py

```diff
@@ -925,9 +925,6 @@ def test_image_update_header(self):
         with fits.open(self.temp("test0.fits")) as hdul:
             assert (orig_data == hdul[1].data).all()
 
-    # The test below raised a `ResourceWarning: unclosed transport` exception
-    # due to a bug in Python <=3.10 (cf. cpython#90476)
```

Comment:
whoops I read this wrong, I thought it was `<3.10`, not `<=3.10`. I revert this but maybe make the message more grepable.